trpcultivate_phenotypes/src/TripalCultivateValidator/TripalCultivatePhenotypesValidatorBase.php

```diff
@@ -201,4 +220,41 @@ public function getConfigAllowNew() {
 
     return $allownew;
   }
+
+  /**
+   * Split or explode a data file line/row values into an array using a delimiter.
+   * 
+   * @param string $row
+   *   A line in the data file.
+   * 
+   * @return array
+   *   An array containing the values extracted from the line after splitting it based
+   *   on a delimiter value.
+   */
+  public static function splitRowIntoColumns(string $row) {
+    // Delimiter:
+
+    // @TODO: use the delimiter getter.
```

Comment:
We cannot use a context variable in a static function. Instead we will have to pass the delimeter in as a variable. I suggest making the delimeter an array to allow us to support multiple types of files :thinking: